landsatxplore/api.py

```diff
@@ -103,6 +107,22 @@ def login(self, username, password):
         self.raise_api_error(r)
         self.session.headers["X-Auth-Token"] = r.json().get("data")
 
+    def login_with_token(self, username, token):
```

Comment:
The old username/password end point is not longer available. I think `login(self, username, password)` should be removed since it'll no longer work.